docs/tutorials/morpheusvm/morpheusvm.md

```diff
@@ -198,11 +203,15 @@ constructing and writing key-value pairs directly from our action.
 
 ## Implementing Storage
 
-We'll break storage down into two components:
+When building a VM with the HyperSDK, the VM developer defines their own state storage
```

Comment:
Explain why the VM needs to implement the storage manager as opposed to just implementing it without justification